planner/core/plan_to_pb_test.go

```diff
@@ -27,7 +27,6 @@ import (
 )
 
 func TestColumnToProto(t *testing.T) {
-	t.Parallel()
```

Comment:
please move serial tests in `serial_test` file.

Reply:
updated. please review again.